### PR TITLE
Typo fix, syntax cleanup, better testing

### PR DESCRIPTION
**Pull request by ccope**
Chefspec convergence actually works now, updated foodcritic, got rid of fc rule violations

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # encoding: UTF-8
 name             'razor_server'
 maintainer       'Myles Steinhauser'
-maintainer_email 'myles.steinhasuer@gmail.com'
```
Comment (masteinhauser) [inline metadata.rb:4]: Heh, nice catch. Thanks!

**Comment by masteinhauser** [comment]
I was surprised why you changed everything from symbols to strings, but I think it ends up actually being better due to the `"razor-ui"` fudging I had to do.

**Comment by ccope** [comment]
I changed it because of FC001 (which is no longer deprecated): https://github.com/acrmp/foodcritic/issues/86#issuecomment-42511855

**Comment by masteinhauser** [comment]
Ah, sweet! Thanks for the link to the discussion, I had no idea this had taken place but it presents some very good points. :thumbsup: 

I am waiting on @jperry to give me Maintainer access to this repo again before I can merge this PR for you.

**Pull request merged**